cli/jobs/pipelines/automl/cli-automl-text-classification-newsgroup-pipeline/components/src_register/register.py

```diff
@@ -0,0 +1,77 @@
+# ---------------------------------------------------------
```

Comment:
Could we please rename the directories to "src" as we are going to add "preprocess.py", "deploy.py" etc steps.

Reply:
Done